plugins/Deflect/utils.h

```diff
@@ -0,0 +1,60 @@
+/* Copyright (c) 2018, EPFL/Blue Brain Project
+ * All rights reserved. Do not distribute without permission.
+ * Responsible Author: Daniel Nachbaur <daniel.nachbaur@epfl.ch>
+ *
+ * This file is part of https://github.com/BlueBrain/Brayns
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License version 3.0 as published
+ * by the Free Software Foundation.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this library; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+#pragma once
+
+#include <brayns/common/utils/enumUtils.h>
+#include <deflect/deflect.h>
+
+#define deflectPixelOp deflectPixelOp
+#define TEXTIFY(A) #A
+
+namespace brayns
+{
+template <>
+inline std::vector<std::pair<std::string, deflect::ChromaSubsampling>> enumMap()
+{
+    return {{"yuv444", deflect::ChromaSubsampling::YUV444},
+            {"yuv422", deflect::ChromaSubsampling::YUV422},
+            {"yuv420", deflect::ChromaSubsampling::YUV420}};
+}
+namespace utils
+{
+inline deflect::View getView(const std::string& name)
```

Comment:
Document somewhere the assumptions of what name encodes

Reply:
the code is enough documentation, no?

Reply:
But where are these names coming from? My fear is that the knowledge is distributed in multiple places, making maintenance more complicated.